### PR TITLE
vesting: Send back pending tokens during stopVesting()

### PR DESCRIPTION
**Pull request by axic**
Alternative to #400.

### CODE REVIEW

**Comment by mijovic** [comment]
I would prefer this one over #400

File: test/token/vesting/ERC20TokenVesting.test.ts [hunk 1]
```diff
@@ -181,14 +181,15 @@ describe("ERC20 Vesting", async () => {
...
-    it("Expected state after stopVesting", async () => {
+    it("Expected state after stopVesting (after period)", async () => {
       const startTime = await blockTimestamp();
       const period = DAY * 30;
       await vesting.startVesting(
         owner,
         user,
         {startTime: startTime, period: period, amount: 30, claimed: 0}
       );
+      await setEvmTime(startTime + period);
```
Comment (axic) [inline test/token/vesting/ERC20TokenVesting.test.ts:192]: This is needed to ensure correct amounts.

**Comment by axic** [comment]
> I would prefer this one over #400

Not sure, while this is simpler, the other fits the model better.

**Comment by mijovic** [comment]
Since we went with #400 closing this one

**Pull request closed without merging**